src/resources/realms.ts

```diff
@@ -53,6 +56,19 @@ export class Realms extends Resource {
     urlParamKeys: ['realm'],
   });
 
+  public partialImport = this.makeRequest<
+    {
+      realm: string;
+      realmRep: PartialImportRealmRepresentation;
```

Comment:
In the [API docs](https://www.keycloak.org/docs-api/15.0/rest-api/index.html#_partialimport) this is simply called `rep`.
```suggestion
      rep: PartialImportRepresentation;
```